code/service/src/main/scala/com/namely/chiefofstate/ServiceBootstrapper.scala

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright 2020 Namely Inc.
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+package com.namely.chiefofstate
+
+import akka.actor.typed.{ActorRef, ActorSystem, Behavior}
+import akka.actor.typed.scaladsl.Behaviors
+import akka.cluster.sharding.typed.scaladsl.{ClusterSharding, Entity}
+import akka.persistence.typed.PersistenceId
+import akka.util.Timeout
+import com.namely.chiefofstate.config.CosConfig
+import com.namely.chiefofstate.plugin.PluginManager
+import com.namely.chiefofstate.telemetry._
+import com.namely.chiefofstate.StartNodeBehaviour.log
+import com.namely.chiefofstate.readside.ReadSideManager
+import com.namely.protobuf.chiefofstate.v1.service.ChiefOfStateServiceGrpc.ChiefOfStateService
+import com.namely.protobuf.chiefofstate.v1.writeside.WriteSideHandlerServiceGrpc.WriteSideHandlerServiceBlockingStub
+import com.typesafe.config.Config
+import io.grpc._
+import io.grpc.netty.NettyServerBuilder
+import io.opentelemetry.api.GlobalOpenTelemetry
+import io.opentelemetry.instrumentation.grpc.v1_5.GrpcTracing
+
+import java.net.InetSocketAddress
+import scala.concurrent.ExecutionContext
+import scala.sys.ShutdownHookThread
+
+/**
+ * This helps setup the required engines needed to smoothly run the ChiefOfState sevice.
+ * The following engines are started on boot.
+ * <ul>
+ *   <li> the akka cluster system
+ *   <li> the akka cluster sharding engine
+ *   <li> the akka cluster management
+ *   <li> loads the various ChiefOfState plugins
+ *   <li> run the required schemas and migration needed
```

Comment:
this is no longer true, right?

Reply:
Yeah we need to add a proper docstring now. It was a copy paste easy oversight

Reply:
Can you fix this?